sentry-test-support/src/main/kotlin/io/sentry/test/assertions.kt

```diff
@@ -0,0 +1,21 @@
+package io.sentry.test
+
+import com.nhaarman.mockitokotlin2.check
+import io.sentry.core.GsonSerializer
+import io.sentry.core.NoOpLogger
+import io.sentry.core.SentryEnvelope
+import io.sentry.core.SentryEvent
+import io.sentry.core.SentryOptions
+
+/**
+ * Verifies is [SentryEnvelope] contains first event matching a predicate.
+ */
+inline fun checkEvent(noinline predicate: (SentryEvent) -> Unit): SentryEnvelope {
```

Comment:
just out of curiosity, could this not live in the sentry-core/test package?

Reply:
Can it be then included in other modules? It wouldn't be straightforward with Maven. Perhaps there is a reasonable way to do it with Gradle?

Reply:
mm good catch, I thought `testImplementation` would be enough, but I think I've seen ways of doing that, just don't remember by heart now.